lesson10/jquery.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const bodyParser= require("body-parser");
+const app = express();
+const gm = require("gm");
+const fs = require("fs");
+
+app.listen(3615);
+app.use(express.static('public'));
+app.use(bodyParser.json({limit:'500mb'}));
+
+app.post('/nameCom', (req,res)=>{
+  fs.writeFile('/home/sya/garageScript/lesson10/public/ncom.txt', ` Name: ${req.body.name} Comment: ${req.body.comment}`)
```

Comment:
ncom would not do if you were working on the LLIP app. make the names direct. nameComment

Reply:
will do!

Reply:
return line \n

Reply:
left out return line intentionally :)